src/tools/kata-ctl/src/ops/volume_ops.rs

```diff
@@ -0,0 +1,300 @@
+// Copyright (c) 2022 Boston University
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use crate::args::{DirectVolSubcommand, DirectVolumeCommand};
+
+use anyhow::{anyhow, Ok, Result};
+use futures::executor;
+use kata_types::mount::{
+    DirectVolumeMountInfo, KATA_DIRECT_VOLUME_ROOT_PATH, KATA_MOUNT_INFO_FILE_NAME,
+};
+use nix;
+use reqwest::StatusCode;
+use safe_path;
+use std::{fs, path::PathBuf, time::Duration};
+use url;
+
+use agent::ResizeVolumeRequest;
+use runtimes;
+
+const TIMEOUT: Duration = Duration::from_millis(200);
```

Comment:
Storage operations may take a lot of time, as a timeout value, 200ms may be too small.